packages/wpcom-checkout/src/postal-code.ts

```diff
@@ -76,6 +76,17 @@ const twoPartPostalCodes: Record<
 		delimiter: ' ',
 		partLength: 3,
 	},
+	US: {
+		length: [ 9 ],
+		delimiter: '-',
+		formatter: ( postalCodeInput: string, delimiter: string ) => {
```

Comment:
Thank you for the PR. One doubt here! Why do we need to write formatter? Something like this `partLength: 5` should work out of the box. It will reduce LOC as well.

Function `defaultFormatter` is also doing same work
https://github.com/Automattic/wp-calypso/blob/ccde1b0b9b44c993d6de513b60ab38458832cf71/packages/wpcom-checkout/src/postal-code.ts#L15-L17

Thanks

Reply:
@amustaque97 Thanks for pointing this out. I've updated this to use `partLength`.